include/rdma/fi_errno.h

```diff
@@ -194,6 +194,7 @@ enum {
 	FI_ENOAV	 = 267, /* Missing or unavailable address vector */
 	FI_EOVERRUN	 = 268, /* Queue has been overrun */
 	FI_ENORX	 = 269, /* Receiver not ready, no receive buffers available */
+	FI_ENOMR	 = 270, /* No more memory registrations available */
```

Comment:
Looks good.  Should the fi_mr man page be updated to reference this error code?

Reply:
FI_ENOMR is not returned to the application. It's only used internally within EFA provider. So I don't think we need to update the man page.